contrib/automation_tests/orbit_symbol_locations.py

```diff
@@ -0,0 +1,62 @@
+"""
+Copyright (c) 2021 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+
+import os
+
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.connection_window import FilterAndSelectFirstProcess, ConnectToStadiaInstance
+from test_cases.symbols_tab import ClearSymbolCache, LoadSymbols
+from test_cases.symbol_locations import AddSymbolLocation, ClearAllSymbolLocations
+
+"""
+Test symbol loading from custom symbol locations.
+
+Before this script is run there needs to be a gamelet reserved and
+"no_symbols_elf" has to be started (can be downloaded from GCS). 
+Test data needs to be available in  "testdata/symbol_location_tests" (part of our git repo).
```

Comment:
I'm unsure about what is a good location for the testdata (GCS vs. github). I think previously the idea was to put "binary blops" into GCS and "human readable data" into github. However storing stuff in github is probaly preferrable in almost every case: The data is under version control and every release of Orbit gets the correct testdata (when it changes). As long as we only touch stuff in github we don't need to worry about version compatibility of the tests.

Reply:
I was following what @florian-kuebler also did for our test capture, he basically had the same reasoning, and while those files are binary blobs, they're reasonably small.

I'm open to storing them in GCS as well, but then it might make sense to introduce some kind of structure that allows us to add some for of simple versioning (could just be an increasing number we append to the file, and change in the Python code...).

Reply:
Yeah, I think putting them in Github is cleaner. BUT, we should keep an eye on the file size, as we would otherwise bloat up the Orbit distributable.